### ISSUE TITLE
bigip_ltm_policy rule condition not working

### ISSUE THREAD

**Issue by myclau** [F5Networks/terraform-provider-bigip#119]
I am trying to create a policy if the uri contain my domain will forward to some pool.
This is my current policy resource 
```tf
resource "bigip_ltm_policy" "policy" {
  name = "${var.poolName}-policy"
  strategy = "first-match"
  requires = ["http"]
  published_copy = "Drafts/${var.poolName}-policy"
  controls = ["forwarding"]
  rule {
    name = "route-${var.poolName}"
    condition {
      http_uri = true
      host = true
      contains = true
      values = ["<mydomain>"]
    }
    action {
      forward = true
      pool = "${bigip_ltm_pool.pool.name}"
    }
 }
  depends_on = ["bigip_ltm_pool.pool"]
}
```

when I `terraform apply`
it will always show up this
```bash
Error: 01071706:3: Policy '/Common/Drafts/<poolname>-policy', rule 'route-<poolname>'; missing operand.

  on main.tf line 121, in resource "bigip_ltm_policy" "policy":
 121: resource "bigip_ltm_policy" "policy" {
```

I try if comment the condition is will create the policy successfully
```tf
resource "bigip_ltm_policy" "policy" {
  name = "${var.poolName}-policy"
  strategy = "first-match"
  requires = ["http"]
  published_copy = "Drafts/${var.poolName}-policy"
  controls = ["forwarding"]
  rule {
    name = "route-${var.poolName}"
    #condition {
    #  http_uri = true
    #  host = true
    #  contains = true
    #  values = ["<mydomain>"]
    #}
    action {
      forward = true
      pool = "${bigip_ltm_pool.pool.name}"
    }
 }
  depends_on = ["bigip_ltm_pool.pool"]
}
```

More info

my current provider settings
```tf
provider "bigip" {
  version = "=0.12.3"
  address = "${var.url}"
  username = "${var.username}"
  password = "${var.password}"
}
```

**Comment by mdditt2000** [comment]
TER-23

**Comment by Pacobart** [comment]
I'm experiencing this same issue. Any update on a fix?

**Comment by davidroberts63** [comment]
Same here. Any updates?

**Comment by papineni87** [comment]
@davidroberts63 This might be fixed in last merge request. Can you clone the latest code and try it ?

**Comment by papineni87** [comment]
Issue is fixed in 1.4 release, closing the issue
